src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -6245,12 +6244,29 @@ test_auto_datakeys (void *unused)
 }
 
 static void
-test_create_encrypted_collection_simple (void *unused)
+_do_cec_test (void (*test) (const char *kmsProvider))
+{
+   // Run the test using the "local" key:
```

Comment:
A bit unclear what this comment is documenting. Is it documenting that local should be tested before AWS?